hack/generate/mesh-auth-policies.sh

```diff
@@ -18,10 +18,15 @@ helm > /dev/null || exit 127
 source "$(dirname "${BASH_SOURCE[0]}")/../lib/metadata.bash"
 
 policies_path="$(dirname "${BASH_SOURCE[0]}")/../lib/mesh_resources/authorization-policies/helm"
-if [ -z "${ISTIO_CHART_VERSION:-}" ]; then
-  chart_version="$(metadata.get project.version)"
-else
-  chart_version="${ISTIO_CHART_VERSION}"
+chart_version="$(metadata.get project.version | grep -Eo '[0-9]+\.[0-9]+')" # grep removes the patch version in semver
```

Comment:
We still need the logic, this is to have the bump flow work on an unpublished version of the chart

Reply:
Ah actually, I see now, we don't need that anymore

Reply:
because we create the branch _before_ generating policies